infra/build/functions/build_lib.py

```diff
@@ -186,6 +186,38 @@ def download_corpora_steps(project_name, testing):
   return steps
 
 
+def download_coverage_data_steps(project_name, latest, bucket_name, testing):
+  """Returns GCB steps to download coverage data for the given project"""
+  steps = []
+  fuzz_targets = _get_targets_list(project_name, testing)
+  if not fuzz_targets:
+    sys.stderr.write('No fuzz targets found for project "%s".\n' % project_name)
```

Comment:
Can you log instead of writing to stderr please?

Reply:
The other functions in `build_lib.py` are writing to stderr, too. Should we change them all?

Reply:
Eh can leave for consistency. We can change them all another time.